src/components/common/Tree/Tree.js

```diff
@@ -5,14 +5,89 @@
   This feature should be ported to graasp-ui. */
 
 /* eslint-disable react/forbid-prop-types */
+/* eslint-disable prefer-arrow-callback */
 import React, { useState } from 'react';
-import TreeView from '@material-ui/lab/TreeView';
+import TreeView from '@mui/lab/TreeView';
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import ChevronRightIcon from '@material-ui/icons/ChevronRight';
-import TreeItem from '@material-ui/lab/TreeItem';
+import TreeItem, { useTreeItem } from '@mui/lab/TreeItem';
 import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import Typography from '@material-ui/core/Typography';
 import CustomTreeItem from './CustomTreeItem';
 
+const CustomContent = React.forwardRef(function CustomContent(props, ref) {
```

Comment:
Is it the same component as defined in CustomTreeItem? Maybe create a whole new file for this component and reuse it.

Reply:
Created `CustomContentTree` component and used for both `Tree` and `CustomTreeItem`